src/hidapi/CMakeLists.txt

```diff
@@ -12,6 +14,10 @@ include_directories(include)
 
 add_library(hidapi STATIC ${HIDAPI_IMPL})
 
+if (UNIX)
```

Comment:
This also breaks other platforms. UNIX matches for Linux and I suppose for Apple as well.